src/ert_shared/ensemble_evaluator/evaluator.py

```diff
@@ -312,7 +311,7 @@ def _run_server(self, loop):
 
     def run(self) -> ee_monitor._Monitor:
         self._ws_thread.start()
-        self._ensemble.evaluate(self._config, self._ee_id)
+        self._ensemble.evaluate(self._config, self.ensemble.id_)
```

Comment:
is `self.ensemble.id_` a necessary parameter?

Reply:
removed in update